odra-macros/src/ast/wasm_parts.rs

```diff
@@ -122,7 +122,13 @@ impl TryFrom<&'_ ModuleImplIR> for CallFnItem {
         let runtime_args_expr: syn::Expr = match module.constructor() {
             Some(f) => {
                 let arg_block = fn_utils::runtime_args_block(&f, wasm_parts_utils::insert_arg_stmt);
-                parse_quote!(let #ident_args = Some(#arg_block))
+                parse_quote!(let #ident_args = {
+                    let env = odra::odra_casper_wasm_env::WasmContractEnv::new_env();
```

Comment:
Is it needed to make ad-hock empty `env`? I think having to create one env per `no_mangle` is enough.